src/geo/map/torque-layer.js

```diff
@@ -56,6 +55,23 @@ var TorqueLayer = LayerModelBase.extend({
     }
   },
 
+  _hasTurboCarto: function () {
+    var currentCartoCSS = this.get('cartocss');
+    var shader = new carto.RendererJS().render(currentCartoCSS);
+    var layer = shader.findLayer({ name: '#layer' });
+
+    if (layer) {
+      var layerShader = layer.getStyle({ property: 1 }, { zoom: 10 });
+
+      var markerFill = layerShader['marker-fill'];
```

Comment:
We could do that yes, not sure if that makes sense in any case, because polygon-fill with torque is not possible (yet), or apply categories (ramp) to line-stroke.